gcloud/bigquery/test__helpers.py

```diff
@@ -70,11 +73,16 @@ def test_w_utc_datetime(self):
 
     def test_w_non_utc_datetime(self):
         import datetime
-        import pytz
+        from gcloud._helpers import UTC
+        from gcloud._helpers import _UTC
         from gcloud.bigquery._helpers import _total_seconds
-        eastern = pytz.timezone('US/Eastern')
-        NOW = datetime.datetime(2015, 7, 28, 16, 34, 47, tzinfo=eastern)
-        EPOCH = datetime.datetime(1970, 1, 1, tzinfo=pytz.utc)
+
+        class CET(_UTC):
```

Comment:
Neither one really did, but I'll add it and fold it into the original commit (along with the default to `pytz` if present)